modules/nixos/cgroups.nix

```diff
@@ -1,5 +1,16 @@
+{ pkgs, ... }:
 {
   nix = {
+    package = pkgs.nix.overrideAttrs (o: {
+      patches = (o.patches or [ ]) ++ [
+        (pkgs.fetchpatch {
+          # https://github.com/NixOS/nix/pull/13135
+          url = "https://github.com/NixOS/nix/commit/7030d2e44fccc6c0fc5f5bc87eb05d7134744131.patch";
```

Comment:
I think we might have to revert this because I saw that the waitpid() will steel the process status in other places.

Reply:
https://github.com/nix-community/infra/pull/1818